cli/mtv_api_tests/run.py

```
@@ -0,0 +1,200 @@
+        "-v",
+        f"--tc=source_provider:{provider_key}",
+        f"--tc=storage_class:{storage_class}",
+        *cluster_args,
+    ]
+    if category != "all":
+        cmd.extend(["-m", category])
+    if test_filter:
+        cmd.extend(["-k", test_filter])
+
+    console.print(f"\n[bold]Running:[/bold] {' '.join(cmd)}\n")
+    result = subprocess.run(cmd)
```

Ruff (0.15.4)

[error] `subprocess` call: check for execution of untrusted input

(S603)